docs/_v15_x/4_archive.md

```diff
@@ -74,11 +74,59 @@ Archive current = archives.getArchives().get(0);
 final List<ArchiveDocument> documents = new ArrayList<>();
 
 while (current.getNextDocuments().isPresent()) {
-current = current.getNextDocuments()
-.map(client::getArchiveDocuments)
-.orElse(new Archive());
+    current = current.getNextDocuments()
+    .map(client::getArchiveDocuments)
+    .orElse(new Archive());
 
-documents.addAll(current.getDocuments());
+    documents.addAll(current.getDocuments());
+}
+
+// This prints to total content of the list of documents
+System.out.println(documents);
+```
+## Archive Document attributes
+
+You can add optional attributes to documents. An attribute is a key/val string-map that describe documents. You can add 
+up to 15 attributes pr. archive document. 
```

Comment:
Kan det vere fint å nevne at både key og value er case-sensitive?

Reply:
```suggestion
up to 15 attributes per archive document. 
```

pr. er ikke en forkortelse på engelsk, mener jeg

Reply:
Ikkje på norsk heller for så vidt, sidan det er like mange tegn. 😆

Reply:
Ja, denne får vi backporte inn i Digipost. Gidder ikke en ny runde nå.